src/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 FetchContent_Declare(
     d3d12translationlayer
     GIT_REPOSITORY https://github.com/microsoft/D3D12TranslationLayer.git
-    GIT_TAG        4e12116d7f6e81e76c20e5afb66417bcd69c16c5
+    GIT_TAG        6ee4db09ce392d13c6df2196abbad4293da579ce

```

Comment:
This will need to be updated again once the changes are merged. Thoughts on just [pointing to the tip of master like in d3d9on12](https://github.com/microsoft/D3D9On12/blob/main/src/CMakeLists.txt#L18-L22) to reduce headache here in the future? I know the official stance is that it's better to point to a specific SHA, but... eh, bit tedius and easy to forget to update.